src/server.c

```diff
@@ -63,6 +63,7 @@
 #include <sys/utsname.h>
 #include <locale.h>
 #include <sys/socket.h>
+#include "allocator_defrag.h"
```

Comment:
All the other includes of our own headers (`"..."`) are above the library includes (`<...>`). Let's follow this style.